### PR TITLE
siyuan-note: Add version 1.4.8

### PR DESCRIPTION
**Pull request by Zuoqiu-Yingyi**

### CODE REVIEW

File: bucket/siyuan-note.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "https://github.com/siyuan-note/siyuan",
```
Comment (pratikpc) [inline bucket/siyuan-note.json:2]: Hey could you follow the usual order when it comes to Scoop packages?

Look at accesser for example

https://github.com/ScoopInstaller/Extras/blob/master/bucket/accesser.json

File: bucket/siyuan-note.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/siyuan-note/siyuan/releases/download/v$version/siyuan-$version-win.exe#/siyuan.7z"
+            },
+            "32bit": {
+                "url": "https://github.com/siyuan-note/siyuan/releases/download/v$version/siyuan-$version-win32.exe#/siyuan.7z"
+            }
+        }
+    }
+}
```
Comment (pratikpc) [inline bucket/siyuan-note.json:47]: Newline missing at the end of the file

File: bucket/siyuan-note.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        }
+    },
+    "shortcuts": [
+        [
+            "siyuan.exe",
+            "SiYuan Note"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/siyuan-note/siyuan/releases/latest",
```
Comment (pratikpc) [inline bucket/siyuan-note.json:34]: Would checkver github not work here?

File: bucket/siyuan-note.json (hunk 1 above)
Comment (pratikpc) [inline bucket/siyuan-note.json:2]: Is the homepage correct?

Isn't this the homepage?

https://b3log.org/siyuan

File: bucket/siyuan-note.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    "version": "1.4.8",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://b3log.org/siyuan/eula.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/siyuan-note/siyuan/releases/download/v1.4.8/siyuan-1.4.8-win.exe#/siyuan.7z",
+            "hash": "2e72af2c6211595ce4e47c2141e029ce53fb7f934d383f9d8f79493ac6a70b61",
+            "pre_install": [
```
Comment (pratikpc) [inline bucket/siyuan-note.json:13]: Does it not work without the pre-install block?

**Review by pratikpc — CHANGES_REQUESTED** [review]

**Comment by rashil2000** [comment]
Where is the relevant package request for this?

Do not ignore the instruction written on the issue template, otherwise this PR will be closed without explanation.

The template has specific instructions for creating a related issue and for ordering of fields in the manifest.

File: bucket/siyuan-note.json (hunk 1 above)
Comment (Zuoqiu-Yingyi) [reply bucket/siyuan-note.json:2]: Thanks for the good check!
But the my original fork has been deleted, I fix the issue in a new PR.
[Pull Request #7254 · ScoopInstaller/Extras](https://github.com/ScoopInstaller/Extras/pull/7254)

File: bucket/siyuan-note.json (hunk 2 above)
Comment (Zuoqiu-Yingyi) [reply bucket/siyuan-note.json:47]: Thanks for the good check!
But the my original fork has been deleted, I fix the issue in a new PR.
[Pull Request #7254 · ScoopInstaller/Extras](https://github.com/ScoopInstaller/Extras/pull/7254)

File: bucket/siyuan-note.json (hunk 3 above)
Comment (Zuoqiu-Yingyi) [reply bucket/siyuan-note.json:34]: Thanks for the good check!
But the my original fork has been deleted, I fix the issue in a new PR.
[Pull Request #7254 · ScoopInstaller/Extras](https://github.com/ScoopInstaller/Extras/pull/7254)

File: bucket/siyuan-note.json (hunk 1 above)
Comment (Zuoqiu-Yingyi) [reply bucket/siyuan-note.json:2]: Thanks for the good check!
But the my original fork has been deleted, I fix the issue in a new PR.
[Pull Request #7254 · ScoopInstaller/Extras](https://github.com/ScoopInstaller/Extras/pull/7254)

File: bucket/siyuan-note.json (hunk 4 above)
Comment (Zuoqiu-Yingyi) [reply bucket/siyuan-note.json:13]: Thanks for the good check!
It not work without the pre-install block

**Pull request closed without merging**